app/src/main/java/lt/getpet/getpet/SplashActivity.kt

```diff
@@ -54,4 +57,14 @@ class SplashActivity : BaseActivity() {
         navigationManager.navigateToMainActivity(this)
         finish()
     }
+
+    private fun showOnBoardingFragment() {
+        PreferenceManager.getDefaultSharedPreferences(this).apply {
```

Comment:
For preferences I recently introduced `AppPreferences` which is injected. https://github.com/GotPet/getpet-android/blob/master/app/src/main/java/lt/getpet/getpet/preferences/AppPreferences.kt#L10

In order to use it declare new boolean preference:
```kotlin
    val onboradingCompleted by lazy {
        preferences. booleanPreference(
                COMPLETED_ONBOARDING_PREF_NAME,
                false
        )
    }
```

And when you can use this preference like in https://github.com/GotPet/getpet-android/blob/bf0a10d048b522815709078897d14f061db49500/app/src/main/java/lt/getpet/getpet/network/ApiHeadersInterceptor.kt#L33